### ISSUE TITLE
Missing visual label for form field

### ISSUE THREAD

**Issue by michellenSD** [logicly-au/docs.iar-dst.online#35]
**'Child Domains Overview' page**
Child Domains Overview
- Search

Steps to Generate PDF report
- Search

<img width="242" alt="Image" src="https://github.com/user-attachments/assets/dafce513-3b3d-4d4a-bc85-b166f6f64bfb" />

_Accessibility Report issue 35_

See https://gh.sdintra.net/IAR-DST/iar-dst-app/issues/288 for report
